packages/router/useRouteParams/index.ts

```diff
@@ -36,8 +36,18 @@ export function useRouteParams<
     transform,
   } = options
 
-  const transformGet = transform && 'get' in transform ? transform.get : transform ?? ((value: T) => value as any as K)
-  const transformSet = transform && 'set' in transform ? transform.set : (value: K) => value as any as T
+  const defaultGet = <T, K>(value: T) => value as any as K
```

Comment:
Since these don't use anything from the current scope, we could move them out of the function to avoid re-creating them on every call

Reply:
Thanks, done.